gateway-js/src/__tests__/build-query-plan.feature

```diff
@@ -1213,7 +1219,8 @@ Scenario: supports multiple root mutations
             "password"
           ],
           "operationKind": "mutation",
-          "operation": "mutation($username:String!$password:String!){login(username:$username password:$password){__typename id}}"
+          "operation": "mutation LoginAndReview_accounts_0($username:String!$password:String!){login(username:$username password:$password){__typename id}}",
+          "operationName": "LoginAndReview_accounts_0"
```

Comment:
Very thankful you chose to add `operationName` in here!

Reply:
👍🏻 I had to in order to provide the operationName query param on the request, because that's probably what observability tools look at (instead of parsing the operation doc).

Reply:
For context as to why i'm glad: it's only recently that we started adding `operationKind` in there, which enabled the router to make Good decisions without needing to — in the same spirit as you're describing — re-parse the `operation`.